### PR TITLE
[CSL-1583] test us

### PR DESCRIPTION
**Pull request by georgeee**

### CODE REVIEW

File: docs/how-to/test-update-system.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+```
+listaddr
+```
+
+You should see something like:
+
+```
+> listaddr
+Available addresses:
+    #0:   1feg76nzCcJrhwX4Z9NXc9iwNwo1UxUdkaF3XwdyFhtDdst (PK: eMrhrnbwqmRmeHNQW6kyrD9gs4sMtUyGRqPuLsiZaaTLhHY4uojgp7imLNX6tsECechdqLUB92APfs6Er1BJv6E)
```
Comment (int-index) [inline docs/how-to/test-update-system.md]: The output of `listaddr` is more verbose after https://github.com/input-output-hk/cardano-sl/pull/1453, the user will see something like this instead:

```
Available addresses:
    #0:   addr:      9d5cGKg5x3Vu3g2APH2QJMjt8WyRKfSE4Nohbz1q9y5FZs51mMWkFCjSt7ps8TwJeSES55jaXhyV3wVJr69Q8CLs2cu4xHpDDx5N
          pk base58: bNfckjryxQLmVh617K6HVPpeKPJWjZsGkNGsfgm64mQQN4Q4sShp73NRDJVXA7pM1Eqt5tcHvTsHsg7kAT7R6bMPix
          pk hex:    2c63dd8d53e64913131b4b8adf7930aff642828f734d64c06bc775863a868c871e9d34286e5aad95515d6cb3a40dd29a3a70ceb345cba6ad6687236be4c922f9
          pk hash:   6dab12ab37cc34b40fc6188969699159c78a9231901b166a897162eb
    #1:   addr:      9d5cHXm3XqMPtyCwKyLEBdqK6bunok466ogy13UbpSggbqaRjkr36n2JfrY3RFvDyT5Qt3Hz6XXgLDepTZVRqDybCZQcJFd2rosi
          pk base58: bNhQZ5x9DGYeV4gLyk1Axa4yp4xzrzZBskXov4Vk7AkrogndKgpkiMLJrNqaCJQAJmyrzAbeq4WtqkYH7mBisM2e2w
          pk hex:    85e7090aafc05f65e7e3774fef2ab12b4c0541f16436fded33472efe41b8c60cc4bce3b4728221f1ea5878777109ca3198a22f9932482eac09bf533e22c0388c
          pk hash:   f4a8b9b0f4671cb2486432619f55309ef4a424a8b3ab191ed7484037
    #2:   addr:      9d5cAS1VZHwNVkdBLVvNPmY9mpa4XuMPrn7DsQe7tmWjAqMDYMYzw54kLfMztQfJPVQN3uJnBCmkUD5f4YKogWXEeeXiBXTY1XqK
          pk base58: bNjEfqa4agzaXii4oEtugN3mMA6UykrfdASYFwNZsJLjoVnhr5FWwyg5ZLHZAqR6XXygPtdbu1bbtJhx1ptddPLWFp
          pk hex:    e169cabc73c70664b4f679a7910941da55ca1736a67513b98bf430c8e3a4490cdb3d70c322fa1943a4e78f90d6207dd6a90a35dd1ccf8dc0d333684d18724ec7
          pk hash:   6056d5a781311a76d5cd7b4f43823de687bb0cba5c4482bf4579fc9d
    #3:   addr:      9d5cKX74agz6Dcu3ZTibzWtR11aciNkSQKh4mFFQMEEdvvbvDwbNgMFK3ZW7Pxx2BERGSNX7G4FUXbYsSZuGNdp8ZCa1EJtQnRYj
          pk base58: bNiDRg1JczbotuRLXEMmsPgpnY5cCE2Dt3oKJVxjp6kgUGgqCgpjctYPJJ3fD3641Zuj9bwJ3zAaMaJkVBGmKFjPn7
          pk hex:    ae52c28bdfcda269ce747b12389bbebb54872f83a81f8b9a2ed69134a9df75a8b8e46ebcfda5562cac6dadd839f132d4f53b6b332053868b296bd740a9872880
          pk hash:   a50d738df567fc3ec39c2e990cd8a95930ec3ccae0b98b2bbaace676
```

Comment (int-index) [reply docs/how-to/test-update-system.md]: (Probably don't copy the output from my comment, it's with different keys)

File: genesis-info/dn.log [hunk 2]
```diff
@@ -0,0 +1,300 @@
+17cc318de Add instruction on update system testing
```
Comment (gromakovsky) [inline genesis-info/dn.log]: Why is it in `genesis-info`?

Comment (georgeee) [reply genesis-info/dn.log]: @gromakovsky b/c it's how last generate script is implemented (devops guys)
Had neither time nor wish to refactor it

**Pull request merged**